reed-solomon-novelpoly/src/field/inc_afft.rs

```diff
@@ -473,13 +473,14 @@ impl AdditiveFFT {
 	}
 }
 
-#[cfg(any(feature = "mock", test))]
-pub mod test_utils {
-	use super::*;
-	use rand::{Rng, SeedableRng};
+#[cfg(test)]
+mod afft_tests {
+	use rand::{SeedableRng, Rng};
+	use crate::f2e16::{Additive, Elt};
+	use reed_solomon_tester::SMALL_RNG_SEED;
 
 	pub fn gen_plain<R: Rng + SeedableRng<Seed = [u8; 32]>>(size: usize) -> Vec<Additive> {
```

Comment:
should this be gated with `#[cfg(all(target_feature = "avx", feature = "avx"))]` to avoid unused warnings?
or maybe the whole tests module instead (not sure about `b_is_not_one` and its purpose)?